examples/fluent-bit/filter-multiline/README.md

```diff
@@ -0,0 +1,51 @@
+### FireLens Example: Concatenate Multiline or Stack trace log messages
+
+Multiline Filter is availble on `aws-for-fluent-bit` >= v2.18.0. It helps to concatenate messages that originally belong to one context but were split across multiple records or log lines. Common examples are stack traces or applications that print logs in multiple lines. More information could be found in [Fluent Bit official doc](https://docs.fluentbit.io/manual/pipeline/filters/multiline-stacktrace).
+
+With this filter, you are able to use Fluent Bit built-in parses with auto detection and multi format support on:
+- go
+- python
+- ruby
+- java (Google Cloud Platform Java stacktrace format)
+As those are built-in, you can directly specify them in a field called `multiline.parser` in `[FILTER]` section.
+ 
+As mentioned in the official doc, [Tail plugin](https://docs.fluentbit.io/manual/pipeline/inputs/tail#multiline-support) is recommended for doing the concatenation on messages split originally by Docker or CRI container engines. We will use it in our example below.
```

Comment:
tail plugin is not very useful for ECS customers, who mostly send logs to stdout. The point of FireLens is to collect that stdout stream and send it to FLuent Bit. So we want an example for that standard use case- which needs the filter.

You can keep some note that multiline works with tail best. But let's try to frame it in a way that would be focused on the customer need. So something like, "If your logs go to files instead of stdout, its recommended to use multiline in tail direclty instead of the filter". 

Your example should mainly be about the filter.